### ISSUE TITLE
Support different strategies for payout

### ISSUE THREAD

**Issue by 0xmad** [privacy-scaling-explorations/maci-platform#391]

**Comment by aguzmant103** [comment]
This is a great idea!! Can you expand on some of those strategies? @0xmad

**Comment by aguzmant103** [comment]
Is this something like:
- Lump-sum
- By milestones
- Streaming

or something different?

**Comment by 0xmad** [comment]
Right, the main idea here is just to implement and integration mechanism that allows to add more strategies for payout. For instance, right now we have QV with max cap but there is no regular QV and Non-QV. And right, other strategies can be added and this is a point of discussion where we need to decide what we want to support.
